client/settings/account-details/index.js

```diff
@@ -5,8 +5,8 @@ import { Button, ExternalLink, Icon } from '@wordpress/components';
 import interpolateComponents from 'interpolate-components';
 import { help } from '@wordpress/icons';
 import styled from '@emotion/styled';
+import SectionStatus from '../section-status';
 import useWebhookStateMessage from './use-webhook-state-message';
-import SectionStatus from './section-status';
```

Comment:
I moved the `section-status` component so it was more accessible. Its now in the `client/settings` hence why this change was necessary. :)